ci/submit-coverage

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Script to submit simplecov coverage report to Code Climate.
+# Only one report can be submitted per commit, so this can only
+# run once pre build, not for each test set.
+
+# The rspec and cucumber reports are all written to repo_root/coverage
+# Simplecov handles locking and merging for parallel tests.
+# The GKE tests can't mount that dir, so they produce a separate report
+# which is merged into repo_root/coverage by this script.
+
+set -eux
+
+dir="coverage"
+bin="cc-test-reporter"
+report="${dir}/.resultset.json"
+
+if [[ ! -e ${report} ]]; then
+    echo "SimpleCov report (${report}) not found"
+    ls -laR ${dir}
+    return 1
+fi
+
+if [[ ! -x ${bin} ]]; then
+    echo "cc-test-reporter binary not found, not reporting coverage data to code climate"
+    ls -laR ${dir}
+    # report is present but reporter binary is not, definitely a bug, exit error.
+    return 1
+fi
+
+gke_report="ci/authn-k8s/output/simplecov-resultset-authnk8s-gke.json"
```

Comment:
nit: this is a const and can probably go up top of the file (or should be within the `if`)